packages/wp-now/README.md

```diff
@@ -4,27 +4,109 @@
 
 ## Getting Started
 
+Before getting started, make sure you have `nvm` installed. If you need to install it first,
+[follow these installation instructions](https://github.com/nvm-sh/nvm#installation).
+
 Follow these steps to build and run `wp-now` locally:
 
 ### Building
 
 To build the project, use the following commands in your terminal:
 
+1. Clone this repository locally
+2. Execute `nvm use`
+3. Execute `yarn install` followed by `yarn build`
+
+### Running
+
+1. Execute `sudo npm install -g nx@latest`. This will make `nx` command globally accessible in the project.
```

Comment:
I didn't need `sudo` on my end. Is it actually needed?

Reply:
I tested without `-sudo` and it also worked so I removed it